tests/data/sample_admission_requests/ad_request_auto_update_approval.json

```diff
@@ -0,0 +1,220 @@
+        },
+        "resource": {
+            "group": "apps",
+            "version": "v1",
+            "resource": "replicasets"
+        },
+        "requestKind": {
+            "group": "apps",
+            "version": "v1",
+            "kind": "ReplicaSet"
+        },
+        "requestResource": {
+            "group": "apps",
+            "version": "v1",
+            "resource": "replicasets"
+        },
+        "name": "test-58d59c69bf",
+        "namespace": "default",
+        "operation": "UPDATE",
+        "userInfo": {
+            "username": "system:serviceaccount:kube-system:deployment-controller",
+            "uid": "ad2a0c14-9ed4-4ef4-8b28-1e66978a7e6c",
+            "groups": [
+                "system:serviceaccounts",
+                "system:serviceaccounts:kube-system",
+                "system:authenticated"
+            ]
+        },
+        "object": {
+            "kind": "ReplicaSet",
+            "apiVersion": "apps/v1",
+            "metadata": {
+                "name": "test-58d59c69bf",
+                "namespace": "default",
+                "uid": "5a112af8-ccc7-4e00-b1bc-f81adc8acffe",
+                "resourceVersion": "8916",
+                "generation": 3,
+                "creationTimestamp": "2023-01-26T10:39:15Z",
+                "labels": {
+                    "app": "test",
+                    "pod-template-hash": "58d59c69bf"
+                },
+                "annotations": {
+                    "deployment.kubernetes.io/desired-replicas": "2",
+                    "deployment.kubernetes.io/max-replicas": "3",
+                    "deployment.kubernetes.io/revision": "2"
+                },
+                "ownerReferences": [
+                    {
+                        "apiVersion": "apps/v1",
+                        "kind": "Deployment",
+                        "name": "test",
+                        "uid": "8d8adb16-b1fc-487d-a778-3b6f556a9050",
+                        "controller": true,
+                        "blockOwnerDeletion": true
+                    }
+                ],
+                "managedFields": []
+            },
+            "spec": {
+                "replicas": 2,
+                "selector": {
+                    "matchLabels": {
+                        "app": "test",
+                        "pod-template-hash": "58d59c69bf"
+                    }
+                },
+                "template": {
+                    "metadata": {
+                        "labels": {
+                            "app": "test",
+                            "pod-template-hash": "58d59c69bf"
+                        }
+                    },
+                    "spec": {
+                        "containers": [
+                            {
+                                "name": "nginx",
+                                "image": "docker.io/library/redis:latest@sha256:325d5a448d8f6c1d30a0a0fb26090343279d4cf23258b26b1745862f332e9479",
```

Comment:
Is that an image that would normally fail validation? Would probably take sth non-existing to make sure this actually tests the feature's approval and not behind the scenes signature approval or test the same thing without monkeypatching env and test that it fails